configs/clip.yml

```diff
@@ -3,6 +3,7 @@ feature_type: 'clip'
 model_name: 'ViT-B/32' # ViT-B/16 RN50x16 RN50x4 RN101 RN50
 batch_size: 1 # Batchsize (only frame-wise extractors are supported)
 extraction_fps: null # For original video fps, leave unspecified "null" (None)
+extraction_total: null # extract a fix number of frames. It is mutually exclusive with "fps"
```

Comment:
It extracts a fix number of frames from video and obtains their features. Did you mean we should add docs to describe it?

Reply:
I was trying to understand what it does. So, regardless of the video duration it will extract the specified number of frames uniformly from the clip? 

What if we ask for more frames than there are in a video?

Reply:
Sorry I didn't express myself clearly. Yes, it will extract a specified number of frames uniformly. If we ask for more frames, some frames will be copied or generated, which depends on `ffmpeg`.